gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/model/GbItem.java

```diff
@@ -0,0 +1,35 @@
+package org.sakaiproject.gradebookng.tool.model;
+
+import java.io.Serializable;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.sakaiproject.grading.api.Assignment;
+import org.sakaiproject.grading.api.CategoryDefinition;
+
+@AllArgsConstructor
+public class GbItem implements Serializable {
```

Comment:
Does GbStatsItem or GbBreakdownItem make more sense?

GbItem may be viewed as a more global vs something a little more specific.

Reply:
I went with GbBreakdownItem. Good idea!